orchestrator/src/main/kotlin/Orchestrator.kt

```diff
@@ -270,7 +272,8 @@ class Orchestrator(
         endpoint: Endpoint<*>,
         header: MessageHeader,
         message: WorkerMessage,
-        status: JobStatus
+        status: JobStatus,
+        issues: List<Issue>? = null
```

Comment:
As I guess we'd also not want to add an empty list of issues, I wonder whether we could instead have `issues: List<Issue> = emptyList()` here, and below do

    if (issues.isNotEmpty()) ortRunRepository.update(job.ortRunId, issues = issues.asPresent())

Reply:
It shouldn't be an issue to add empty lists since we always add empty issues if there are none (see [here](https://github.com/eclipse-apoapsis/ort-server/blob/main/dao/src/main/kotlin/repositories/DaoOrtRunRepository.kt#L77) and [here](https://github.com/eclipse-apoapsis/ort-server/blob/main/dao/src/main/kotlin/Database.kt#L212-L213)). So, it's a matter of preference whether using `issues: List<Issue>? = null` or `List<Issue> = emptyList()`.